src/lm.jl

```diff
@@ -185,6 +185,31 @@ function adjr2(obj::LinearModel)
     1 - (1 - r²(obj))*(n-1)/(n-p)
 end
 
+struct FStatistic
+    nobs::Float64
+    ndims::Int64
+    f::Float64
+    fprob::Float64
+end
+
+function StatsModels.fstatistic(obj::LinearModel)
+    rss = deviance(obj)
+    tss = nulldeviance(obj)
+
+    n = nobs(obj)
+    p = dof(obj) - 2 # -2 for dispersion parameter and intercept
+    fstat = ((tss - rss) / p) / (rss / (n - p - 1))
```

Comment:
```suggestion
    fstat = ((tss - rss) * (n - p - 1)) / (rss * p)
```

Reduce the number of divisions with algebra :smile: